stack.yaml

```diff
@@ -2,6 +2,7 @@ resolver: nightly-2019-09-21
 packages:
 - .
 extra-deps:
+- aeson-1.4.6.0
```

Comment:
Why?

Reply:
Oh, was cherry-picking from #801. It was going to be a temporary change in the PR. Gone now, due to merge from `master`